include/triton/common/logging.h

```diff
@@ -125,7 +129,10 @@ class Logger {
   // Flush the log.
   void Flush();
 
+  static const std::array<const char*, Level::kINFO + 1> LEVEL_NAMES;
```

Comment:
Should have defined `Level:kEND` then use it instead of `Level::kINFO + 1`, easy to miss if someone adds new log level and doesn't change there.